src/web/mjs/engine/Connector.mjs

```diff
@@ -449,7 +462,7 @@ export default class Connector {
      *
      */
     fetchJSON( request, retries ) {
-        retries = retries || 0;
+        retries = retries || this.retries;
```

Comment:
https://github.com/manga-download/hakuneko/pull/3232#discussion_r600717609
and here